lightkurve/targetpixelfile.py

```diff
@@ -480,209 +480,14 @@ def to_fits(self, output_fn=None, overwrite=False):
-            cadence=lc.cadenceno,
-            quality_code=lc.quality,
-            quality=np.array(qual_strings)))
-
-        # Provide extra metadata in the title
-        if self.mission == 'K2':
-            title = "Quicklook lightcurve for EPIC {} (K2 Campaign {})".format(
-                self.keplerid, self.campaign)
-        elif self.mission == 'Kepler':
-            title = "Quicklook lightcurve for KIC {} (Kepler Quarter {})".format(
-                self.keplerid, self.quarter)
-        elif self.mission == 'TESS':
-            title = "Quicklook lightcurve for TIC {} (TESS Sector {})".format(
-                self.ticid, self.sector)
-        else:
-            title = "Quicklook lightcurve for target {}".format(self.targetid)
-
-        # Figure 1 shows the lightcurve with steps, tooltips, and vertical line
-        fig1 = figure(title=title, plot_height=300, plot_width=600,
-                      tools="pan,wheel_zoom,box_zoom,save,reset")
-        fig1.yaxis.axis_label = ytitle
-        fig1.xaxis.axis_label = 'Time [{}]'.format(lc.time_format.upper())
-        fig1.step('time', 'flux', line_width=1, color='gray', source=source,
-                  nonselection_line_color='gray', mode="center")
-
-        r = fig1.circle('time', 'flux', source=source, fill_alpha=0.3, size=8,
-                        line_color=None, selection_color="firebrick",
-                        nonselection_fill_alpha=0.0, nonselection_line_color=None,
-                        nonselection_line_alpha=0.0, fill_color=None,
-                        hover_fill_color="firebrick", hover_alpha=0.9,
-                        hover_line_color="white")
-
-        fig1.add_tools(HoverTool(tooltips=[("Cadence", "@cadence"),
-                                           ("Time ({})".format(lc.time_format.upper()), "@time{0,0.000}"),
-                                           ("Time (ISO)", "@time_iso"),
-                                           ("Flux", "@flux"),
-                                           ("Quality Code", "@quality_code"),
-                                           ("Quality Flag", "@quality")],
-                                 renderers=[r],
-                                 mode='mouse',
-                                 point_policy="snap_to_data"))
-        # Vertical line to indicate the cadence shown in Fig 2
-        vert = Span(location=0, dimension='height', line_color='firebrick',
-                    line_width=4, line_alpha=0.5)
-        fig1.add_layout(vert)
-
-        # Figure 2 shows the Target Pixel File stamp with log screen stretch
-        if self.mission in ['Kepler', 'K2']:
-            title = 'Pixel data (CCD {}.{})'.format(self.module, self.output)
-        elif self.mission == 'TESS':
-            title = 'Pixel data (Camera {}.{})'.format(self.camera, self.ccd)
+    def interact(self, classic=False, lc=None, notebook_url='localhost:8888'):
+        from . import interact
+        if not classic:
```

Comment:
Is there a good reason to keep the classic version?  If not, I'd keep the code base simple and only offer the new version!